client/cache/error_code.go

```diff
@@ -123,6 +123,14 @@ var InvalidSsdbCacheValue = berror.DefineCode(4002022, moduleName, "InvalidSsdbC
 SSDB cache only accept string value. Please check your input.
 `)
 
+var InvalidInitParameters = berror.DefineCode(4002024, moduleName, "InvalidInitParameters", `
+Invalid init cache parameters.
+`)
+
+var PersistCacheFailed = berror.DefineCode(4002025, moduleName, "PersistCacheFailed", `
+StoreFunc execute failed.
```

Comment:
Failed to execute the StoreFunc.
Please check the log to make sure the StoreFunc works for the specific key and value

Reply:
@hookokoko please change this.

Reply:
All right

Reply:
Sorry, I don’t understand what you mean.  Do you mean that there is no need to define error codes？